packages/amplify-util-node-template-provider/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "amplify-util-node-template-provider",
+  "version": "1.0.0",
+  "description": "Node JS templates supplied by the Amplify Team",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf ./lib"
+  },
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/inquirer": "^6.5.0"
```

Comment:
We're on inquirer 7.0.3 and I don't see it in dependencies, it should be there if we're using it, unfortunately types are not updated but it seems are compatible.

Reply:
fixed